pkgs/top-level/php-packages.nix

```diff
@@ -212,12 +215,25 @@ lib.makeScope pkgs.newScope (self: with self; {
       sourceRoot = "php-${php.version}/ext/${name}";
 
       enableParallelBuilding = true;
-      nativeBuildInputs = [ php.unwrapped autoconf pkg-config re2c ];
+      nativeBuildInputs = [
```

Comment:
This change would be useful in `build-pecl.nix` for me as well! Not sure if it would apply generally to all `buildPecl` built packages, but right now I have to specify: 
```
nativeBuildInputs = [ pkg-config libgit2 (php.withExtensions ({all, ...}: with all; [tokenizer])) ];
```

for my own package.

Setting `internalDeps` doesn't seem to do anything with `buildPecl`. How am I supposed to use the `ext` directory?